frontend/src/components/profile/components/profile-info/add-section/add-section.scss

```diff
@@ -0,0 +1,51 @@
+.add-section {
```

Comment:
Try to use bootstrap classes instead of custom styles. You can remove 80% of these styles and use bootstrap.